backend/models/all.py

```diff
@@ -179,19 +199,19 @@ class Post(Base):
     course_id = Column(
         PUUID,
         ForeignKey(
-            "public.courses.id", ondelete="CASCADE", name="posts_course_id_fkey"
+            "public.courses.id", ondelete="CASCADE"
         ),
         nullable=False,
     )
 
     # Regular fields
     title = Column(Text)
     content = Column(Text)
-    status = Column("status", Enum("poststatus", schema="public"), nullable=True)
+    status = Column("status", Enum(PostStatus, name="poststatus", schema="public"), nullable=True)
     applied_at = Column(DateTime(timezone=True), server_default=func.now())
     created_by = Column(
         PUUID,
-        ForeignKey("public.profiles.id", name="posts_created_by_fkey"),
+        ForeignKey("public.profiles.id"),
```

Comment:
do not delete FK names. it will make duplicates. in fact check ur table definition on supabase you will find many duplications - thx @bcw117 for catching it